main.go

```diff
@@ -180,8 +185,15 @@ func visit(url *url.URL) {
 	}
 
 	t5 := time.Now() // after read response
-	bodyMsg := readResponseBody(resp)
+	bodyRead := readResponseBody(req, resp)
+	var bodyMsg string
+	if bodyRead {
+		bodyMsg = color.CyanString("Body read")
```

Comment:
Please handle the message inside `readResponseBody` there shouldn't need to be a boolean to switch this off or on. If `readResponseBody` returns `""` then 211 will elide printing a message.